packages/react-native-bridge/android/react-native-bridge/build.gradle

```
@@ -0,0 +1,180 @@
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-linear-gradient', readHashedVersion('../../../react-native-editor/package.json', 'react-native-linear-gradient', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-slider', readHashedVersion('../../../react-native-editor/package.json', '@react-native-community/slider', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-get-random-values', readHashedVersion('../../../react-native-editor/package.json', 'react-native-get-random-values', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-masked-view', readHashedVersion('../../../react-native-editor/package.json', '@react-native-community/masked-view', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-gesture-handler', readHashedVersion('../../../react-native-editor/package.json', 'react-native-gesture-handler', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-screens', readHashedVersion('../../../react-native-editor/package.json', 'react-native-screens', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-safe-area-context', readHashedVersion('../../../react-native-editor/package.json', 'react-native-safe-area-context', 'dependencies')))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-reanimated', readHashedVersion('../../../react-native-editor/package.json', 'react-native-reanimated', 'dependencies')))
+   } else {
+        api 'com.facebook.react:react-native:+'
+        api project(':@wordpress_react-native-aztec')
+        api project(':react-native-linear-gradient')
+        api project(':react-native-svg')
+        api project(':react-native-video')
+        api project(':@react-native-community_slider')
+        api project(':react-native-get-random-values')
+        api project(':@react-native-community_masked-view')
+        api project(':react-native-gesture-handler')
+        api project(':react-native-screens')
+        api project(':react-native-safe-area-context')
+        api project(':react-native-reanimated')
+        debugRuntimeOnly project(":hermes-debug")
+        releaseRuntimeOnly project(":hermes-release")
+   }
+}
+
+preBuild.doFirst {
+    if (willPublishReactNativeBridgeBinary) {
+        // Verify that the bundle is correctly copied before building to publish. This will be automatically done
+        // from the shell script that's used to publish the binaries, but this will serve as an extra check if anyone
+        // tries to manually publish the binary.
+        def assertionMessage = "Bundle file is missing. Please ensure that you run `npm install` from the root Gutenberg project and copy the bundle file to react-native-bridge/builds/assets folder"
```

Wise move adding this, and thank you for the comment explaining it 👍 

I think the message should tell users to run the command to build the JS bundle as well, right?

```suggestion
        def assertionMessage = "Bundle file is missing. Please ensure that you run `npm install && npm run bundle:android` from the root Gutenberg project and copy the bundle file to the react-native-bridge/builds/assets folder"
```

---

Addressed in https://github.com/oguzkocer/gutenberg/commit/4c67a6c540ba2fda7776c9e3b93d7f8381bf80d4